openvdb/openvdb/tree/RootNode.h

```diff
@@ -2629,6 +2668,18 @@ RootNode<ChildT>::addChild(ChildT* child)
     return true;
 }
 
+#if OPENVDB_ABI_VERSION_NUMBER >= 10
+template<typename ChildT>
+inline void
+RootNode<ChildT>::setOrigin(const Coord &origin)
+{
+    mOrigin = origin;
+    if (mOrigin != Coord(0,0,0)) {
+        OPENVDB_THROW(ValueError, "RootNode::setOrigin: non-zero offsets are still not supported");
```

Comment:
Maybe we use "currently" instead of "still"? Or otherwise, we could add "can you believe it?" to the end? :)